releng/jenkins/sign-and-deploy/Jenkinsfile

```diff
@@ -0,0 +1,272 @@
+pipeline {
+  agent {
+    kubernetes {
+      label 'releng-build-artifacts-for-release-' + env.BUILD_NUMBER
```

Comment:
Do we want to change the label to 'sign-and-deploy-' + env.BUILD_NUMBER to reflect the job name correctly?

Reply:
yes, will do